src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -553,7 +553,9 @@ def emit(level, message, exc_info=None, **extras):
               'path': path_name,
               'line': line_number,
               'method': method_name
-          }
+          },
+          'release': os.environ.get('CLUSTERFUZZ_RELEASE', 'prod'),
```

Comment:
nit: let's create a helper for this too. e.g. `get_clusterfuzz_release()` in `utils`.

Reply:
I noticed the release/docker_image fields are not being logged over to GCP. This requires some debugging